coordinator/Dockerfile

```diff
@@ -1,20 +1,32 @@
-FROM openjdk:22-ea-17-slim-bookworm
+FROM ubuntu as jar
```

Comment:
Why uses `ubuntu` as the base image? I thought the PR would be using the same image as the [one](https://github.com/Consensys/linea-monorepo/blob/43782deca24f9fc90dda8e3e5dafa8fd64d39b33/transaction-exclusion-api/Dockerfile#L2) used in transaction-exclusion-api dockerfile, and we should also avoid using `openjdk:22-ea-17-slim-bookworm` as it has vulnerbilities listed in docker hub.

Reply:
Additionally, please use images with versions

Reply:
@jonesho I tried the image you mentioned and am getting errors with the native libraries for Go it seems. On going issue described here https://github.com/golang/go/issues/13492